frontend/tablet/src/screens/room.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { View, Text, StyleSheet } from 'react-native'
+
+const App : React.FC = () => {
```

Comment:
Rename `App` -> `RoomScreen`. But I think we will have a `RoomListScreen` we talked about. Then this makes sense to be called `RoomDetailScreen` instead. Since there's no way for me to tell which one you meant to add here, use the appropriate one.

Also, make it the file name. Component files in react start with a capital letter and if it holds more than 1 component, then make it the name of the "main one" from that file.

Reply:
Names fixed in the last commit.